lib/timeline/Timeline.js

```diff
@@ -492,13 +492,13 @@ Timeline.prototype.focus = function(id, options) {
       // Double check we ended at the proper scroll position
       setFinalVerticalPosition();
 
-      // Let the redraw settle and finalize the position.      
+      // Let the redraw settle and finalize the position.
       setTimeout(setFinalVerticalPosition, 100);
     };
 
     // calculate the new middle and interval for the window
     var middle = (start + end) / 2;
-    var interval = Math.max(this.range.end - this.range.start, (end - start) * 1.1);
+    var interval = (end - start) * 1.1;
```

Comment:
`Math.max` ensured a zoom out, but never a zoom in, which is odd. make the behavior more consistent by just snapping to the item with 5% buffer on each side.